crates/editor/src/editor_settings.rs

```diff
@@ -502,6 +518,11 @@ pub struct EditorSettingsContent {
     ///
     /// Default: true
     pub drag_and_drop_selection: Option<bool>,
+
+    /// How to render LSP `textDocument/documentColor` colors in the editor.
+    ///
+    /// Default: `DocumentColors::Underline`
```

Comment:
This should be change to inlay I believe 
```suggestion
    /// Default: `DocumentColorsRenderMode::Inlay`
```